fiducial_slam/README.md

```diff
@@ -47,6 +47,8 @@ in rviz for debugging.
 * `/fiducial_pose` a topic of `geometry_msgs/PoseWithCovarianceStamped` containing
 the computed pose.
 
+*  /fiducial_map  fiducial_slam/FiducialMapEntryArray
```

Comment:
Please format this like the other lines.

`fiducial_map` a topic of type `fiducial_slam/FiducialMapEntryArray`